go.mod

```diff
@@ -5,13 +5,15 @@ go 1.13
 require (
 	github.com/Masterminds/semver v1.4.2
 	github.com/aws/aws-sdk-go v1.20.15
+	github.com/coreos/etcd v3.3.13+incompatible
```

Comment:
This one is not used anywhere and I don't have any idea how it ended up there...

Reply:
try running `go mod tidy`

Reply:
I tried (and pushed a commit), but there is still an entry in `go.sum`.